cobalt/watchdog/watchdog.cc

```diff
@@ -541,35 +544,32 @@ bool Watchdog::Ping(const std::string& name, const std::string& info) {
 
 std::string Watchdog::GetWatchdogViolations(
     const std::vector<std::string>& clients, bool clear) {
```

Comment:
Side comment: When is clear ever false? I don't see any real uses of it. Why do we need the "clear" argument at all? @sherryzy

Reply:
Well, this should be a @briantting 's question. I don't think we can set this clear flag from the outside Cobalt. But in the cobalt code, we can definitely have the freedom to set the flag.
So either we clean this flag up, or we should handle the case when `clear` is false to be consistent.

Reply:
clear is false occasionally in watchdog_test, the clear flag is never exposed / available to kabuki though

Reply:
Should we delete it? No one is using it. It's also dangerous to use if those violations don't get fetched. Separate cl maybe?

Reply:
separate cl, for now were keeping the clear in this cl